TransferMethod/Sources/View/Widget/SelectionWidget.swift

```diff
@@ -100,6 +100,7 @@ final class SelectionWidget: AbstractWidget {
     }
 
     private func updateLabelFieldValue(_ option: HyperwalletFieldSelectionOption) {
```

Comment:
This method is called very 1st time when page loads as well in `setupLayout(field: HyperwalletField)`. So this will mark the field updated even if user hasn't changed the selection. So I suggest to move this logic `isValueUpdated = true` to in here 
`tableView.selectedHandler = { option in
            self.updateLabelFieldValue(option)
            if self.isValid() {
                self.hideError()
            } else {
                self.showError()
            }
        }`

Reply:
Updated